doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md

```diff
@@ -155,10 +155,105 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
 ###<a name="section3.2"></a>Using a specific configuration
 As seen above, the default configuration distributed with the image is tied to certain values that may not be suitable for you tests. Specifically:
 
-* It only works for building historical context data in MySQL.
-* The endpoint for MySQL is `mysql`.
-* The logging level is `INFO`.
-* The logging appender is `console`.
+* MySQL:
+  * It only works for building historical context data in MySQL.
+  * The user for MySQL is `mysql`.
+  * The pass for MySQL is `mysql`.
+  * The logging level is `INFO`.
+  * The logging appender is `console`.
+* Mongo:
+  * It only works for building historical context data in Mongo.
+  * The user for Mongo is `mongo`.
+  * The pass for Mongo is `mongo`.
+  * The logging level is `INFO`.
+  * The logging appender is `console`.
+* STH:
+  * It only works for building historical context data in STH.
+  * The user for STH is `mongo`.
+  * The pass for STH is `mongo`.
+  * The logging level is `INFO`.
+  * The logging appender is `console`.
+* CKAN:
+  * It only works for building historical context data in CKAN.
+  * The endpoint for CKAN is `iot-ckan`.
```

Comment:
I would add in this and all the other lines something like... "The endpoint for CKAN is This can be changed through the `CYGNUS_CKAN_HOST` environment variable.".